website/static/js/profile.js

```diff
@@ -673,6 +673,8 @@ SocialViewModel.prototype.serialize = function() {
             return value;
         }
     );
+    var twitter = serializedData.twitter;
+    serializedData.twitter = twitter.replace(/@/g, '');
```

Comment:
It would be good to avoid special-casing in this method. Instead, could you modify the regex in `socialRules.twitter` to exclude leading `@` signs?